pkg/gardenlet/controller/managedseed/managedseed_actuator.go

```diff
@@ -675,7 +680,17 @@ func ensureGardenletEnvironment(deployment *seedmanagementv1alpha1.GardenletDepl
 	return deployment
 }
 
-func (a *actuator) prepareGardenClientConnectionWithBootstrap(ctx context.Context, shootClient kubernetes.Interface, gcc *configv1alpha1.GardenClientConnection, objectMeta metav1.ObjectMeta, bootstrap seedmanagementv1alpha1.Bootstrap) (string, error) {
+func (a *actuator) prepareGardenClientConnectionWithBootstrap(
+	ctx context.Context,
+	shootClient kubernetes.Interface,
+	gcc *configv1alpha1.GardenClientConnection,
+	managedSeed *seedmanagementv1alpha1.ManagedSeed,
+	seed *gardencorev1beta1.Seed,
+	bootstrap seedmanagementv1alpha1.Bootstrap,
+) (
+	string,
+	error,
+) {
```

Comment:
Nit: How about fewer lines of code?
```suggestion
) (string, error) {
```

Reply:
If a function's input parameters are each put into a dedicated line then - for symmetry reasons and to make it better readable - I typically also put the output parameters into a dedicated line.

Reply:
OK, not insisting, I tend to split if the line is too long, but not otherwise, but obviously this is pure preference.